$NetBSD: patch-src_CMakeLists.txt,v 1.1 2019/08/11 16:47:52 fox Exp $

Do not create a include/decaf/decaf.

Removed the trailing decaf so that the directory containing the
includes is include/decaf.

--- src/CMakeLists.txt.orig	2019-05-05 22:55:13.720148509 +0000
+++ src/CMakeLists.txt
@@ -119,7 +119,7 @@ if(ENABLE_SHARED)
 endif()
 
 install(DIRECTORY ${GSOURCE_PATH}/include/
-	DESTINATION include/decaf
+	DESTINATION include
 	FILES_MATCHING PATTERN "*.h*"
 	PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ
 )
